src/utils/useStylesInterface.js

```diff
@@ -0,0 +1,15 @@
+import { useContext } from 'react';
+
+import WithStylesContext from '../WithStylesContext';
```

Comment:
even importing this will fail in React < 16.3, since it doesn't conditionally check if `createContext` exists